### DISCUSSION TITLE
I have to make JSPanel in Browser Range

### DISCUSSION THREAD

**Discussion by kaushikp707** [Flyer53/jsPanel4#198] [Q&A]
I have JSPanel Popup i have to make that panel in browser range

**Comment by Flyer53** [comment]
I don't have the slightest idea what you want to ask. Please try to explain your request in an understandable words.

**Reply by kaushikp707** [reply]
i have one JSPanel popup which is draggable when i drag this panel it over lap on parent element and after that i am not able to see again this panel i have to refresh page

**Reply by Flyer53** [reply]
Did you check parameter `containment` of[ option.dragit](https://jspanel.de/#options/dragit) already?

**Reply by kaushikp707** [reply]
no i have no idea how to use it can you give example for that

**Reply by Flyer53** [reply]
Then I suggest you take a look at the docs first. There it's explained, including an example.

https://jspanel.de/#options/dragit

Or simply try for example:
```javascript
jsPanel.create({
  dragit: {
    containment: [60, 20, 20, 20]
  }
});
```
